app/locales/eula/eula_en.js

```diff
@@ -0,0 +1,19 @@
+export const eula_en = `## EULA and legal things 
+
+Text here 
+
+yes its text 
+
+many lines 
```

Comment:
Just en.js

Can this be a raw .MD file? That'd be even better then it will render previews on GitHub

Reply:
We would need an extra markdown renderer package to render it I believe

Reply:
There is a markdown renderer built in here. But I am unsure how to get the eula into md sadly